definitions/divorce/json/CaseEventToFields/CaseEventToFields-aos-offline-resp-nonprod.json

```diff
@@ -0,0 +1,373 @@
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "solAOSSOT",
+    "PageFieldDisplayOrder": 21,
+    "DisplayContext": "MANDATORY",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "respondentSolicitorRepresented=\"Yes\"",
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorName",
+    "PageFieldDisplayOrder": 22,
+    "DisplayContext": "MANDATORY",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "respondentSolicitorRepresented=\"Yes\"",
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorCompany",
+    "PageFieldDisplayOrder": 23,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "respondentSolicitorRepresented=\"Yes\"",
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorAddress",
+    "PageFieldDisplayOrder": 24,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "respondentSolicitorRepresented=\"Yes\"",
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorEmail",
+    "PageFieldDisplayOrder": 25,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "respondentSolicitorRepresented=\"Yes\"",
+    "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "EnterAOSOffline_Respondent",
+    "CaseFieldID": "D8RespondentSolicitorPhone",
+    "PageFieldDisplayOrder": 26,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "aosdataentry",
+    "PageLabel": "Enter scanned form case data",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "respondentSolicitorRepresented=\"Yes\"",
+    "ShowSummaryChangeOption": "Yes"
+  }
+]
```

Comment:
Added definitions for AOS respondent in a separate **-nonprod.json** file. This helped me to isolate the changes i needed to make focus on just the required changes and not have to ward through lines of json file.

Reply:
Is your code in COS also feature toggled?

When cos code is promoted to prod and not feature toggled off it may be a problem.